errata.md

```diff
@@ -8,9 +8,14 @@ page 50 (51):
 
 - As we can see, the result looks somewhat similar for lossf_a is $\hat \theta = 0.32$
 
+page 53 (54):
+
+- As I am not a [not a] ethologist [...]
+
 page 59 (60):
 
 - second paragraph: "I(t')s the theoretical distribution [...]"
+- second paragraph: The sentence "Is the theoretical distribution itself the one without a defined mean." is somehow unrelated to the context. Maybe rephrase.
```

Comment:
Is related to the previous sentence, but I will follow your suggestion and rephrase this in a future text.